### PR TITLE
Brand avatar #1184

### PR DESCRIPTION
**Pull request by Sylvain-Wehrum**
- new brand-avatar component + demo page
- fixed: color and pointer of letter-avatar component
- fixed: node-sass version causing demo app to not start

### CODE REVIEW

File: src/clr-addons/brand-avatar/brand-avatar.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
... 330.694839,109.5 C330.694839,127.856734 324.871122,144.834138 314.997771,158.712527 M452.339866,158.712527 C442.473202,144.834138 436.642799,127.856734 436.642799,109.5 C436.642799,91.1455053 442.473202,74.1658623 452.339866,60.2897121 C462.208759,74.1658623 468.034705,91.1455053 468.034705,109.5 C468.034705,127.856734 462.208759,144.834138 452.339866,158.712527 M521.012028,0 C494.978031,0 471.088089,9.18508445 452.342095,24.4853482 C433.593873,9.18508445 409.70393,0 383.672162,0 C357.638165,0 333.748222,9.18508445 314.997771,24.4831091 C296.254006,9.18508445 272.364064,0 246.327838,0 C220.293841,0 196.403899,9.18508445 177.657905,24.4831091 C158.916369,9.18508445 135.021969,0 108.990201,0 C48.7961934,0 0,49.0244366 0,109.5 C0,169.975563 48.7961934,219 108.990201,219 C135.021969,219 158.916369,209.814916 177.657905,194.512413 C196.403899,209.814916 220.293841,219 246.327838,219 C272.364064,219 296.254006,209.814916 314.997771,194.512413 C333.748222,209.814916 357.638165,219 383.672162,219 C409.70393,219 433.593873,209.814916 452.342095,194.512413 C471.088089,209.814916 494.978031,219 521.012028,219 C581.206035,219 630,169.975563 630,109.5 C630,49.0244366 581.206035,0 521.012028,0"
```
Comment (Wallabeng) [inline src/clr-addons/brand-avatar/brand-avatar.html:24]: pls move those icons to our custom icons in "icons" folder and reference it with clr-icon component

**Review by Wallabeng — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -89,7 +89,7 @@
     "karma-scss-preprocessor": "4.0.0",
     "lite-server": "2.6.1",
     "ng-packagr": "11.2.4",
-    "node-sass": "6.0.1",
+    "node-sass": "~5.0.0",
```
Comment (Wallabeng) [inline package.json:92]: node-sass is removed completely now, so you just need to rebase here

**Review by Wallabeng — CHANGES_REQUESTED** [review]

**Comment by Sylvain-Wehrum** [comment]
moved svgs to icons and prefixed with brand- since seat icon already exists (not the brand, the seat for sitting)

**Comment by Sylvain-Wehrum** [comment]
image

File: package-lock.json [hunk 3]
```diff
@@ -172,6 +172,12 @@
             "source-map": "^0.5.0"
           },
           "dependencies": {
+            "lodash": {
+              "version": "4.17.21",
+              "resolved": "https://repository.porscheinformatik.com/repository/npm-public/lodash/-/lodash-4.17.21.tgz",
```
Comment (Wallabeng) [inline package-lock.json:177]: this is a public repository, so no internal porsche links should be used.
if you don't change anything in package.json, just use "npm ci" to install dependencies

File: src/clr-addons/brand-avatar/brand-avatar.scss [hunk 4]
```diff
@@ -0,0 +1,28 @@
+@include exports('brand-avatar.clr-addons') {
+  .avatar-container {
```
Comment (Wallabeng) [inline src/clr-addons/brand-avatar/brand-avatar.scss:2]: pls nest everything inside .brand-avatar and set this class to the component selector via host binding inside @Component.
this ensures that the css doesn't get polluted.
see e.g. pager.ts and the class "pager"

**Review by Wallabeng — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 3 above)
Comment (Sylvain-Wehrum) [reply package-lock.json]: done

File: src/clr-addons/brand-avatar/brand-avatar.scss (hunk 4 above)
Comment (Sylvain-Wehrum) [reply src/clr-addons/brand-avatar/brand-avatar.scss:2]: done

**Comment by Sylvain-Wehrum** [comment]
rebased to current master & squashed all previous commits (so we see the exact differences) ... verifier still fails (no more differences in package-lock.json)

File: src/clr-addons/clr-addons.module.ts [hunk 5]
```diff
@@ -29,6 +29,7 @@ import { ClrTreetableModule } from './treetable/treetable.module';
 import { ClrViewEditSectionModule } from './view-edit-section/view-edit-section.module';
 import { ClrHistoryModule } from './history/history.module';
 import { ClrAutocompleteOffModule } from './autocomplete-off/autocomplete-off.module';
+import { ClrBrandAvatarModule } from './brand-avatar';
```
Comment (Wallabeng) [inline src/clr-addons/clr-addons.module.ts:32]: the maximum call stacks error originates from here. Just reference the module file directly and its gone:
import { ClrBrandAvatarModule } from './brand-avatar/brand-avatar.module';

**Review by Wallabeng — CHANGES_REQUESTED** [review]

**Review by Wallabeng — APPROVED** [review]

**Pull request merged**